EpsilonAuxBMS/AuxBMSTests.adoc

```diff
@@ -0,0 +1,83 @@
+= Aux BMS Tests
+
+== HeartBeat Test
+----
+Elapsed 1000ms{
+	Expect 1 Heartbeat CAN messages (0x720)
+}
+----
+
+== Aux Test
+----
+Elapsed 1000ms{
+	Expect 10 Aux CAN messages (0x721)
+}
+----
+
+== CAN Values to Test
+[options="header"]
+|===
+| Input Type | CAN_ADDRESS | DATA_INDEX | TIME_FRAME
+| PrechargeState |  0x721  | 0 |  100ms 
+| AuxVoltage |  0x721 |  0 |  100ms |
+|===
+
+== Precharge Test
+----
+Send (PRECHARGE_CAN_MESSAGE(0x721)[DATA_INDEX])
+Within 100ms {
+    Expect (PRECHARGE_STATUS_CAN_MESSAGE(0x721)[DATA_INDEX])
+}
+----
+
+== AuxVoltage Test
+----
+Within 100ms{
+    Expect (CURRENT_VOLTAGE_AUX_BATTERY_CAN_MESSAGE(0x721)[DATA_INDEX])
+}
+----
+
+== Contactor Charge Test
+----
+Within 100ms{
+	Expect (CONTACTOR_CURRENT_CAN_MESSAGE(0x721)[DATA_INDEX])
+}
+----
+
+== Contactor State Test
+----
+Within 100ms{
+	Expect (CONTACTOR_STATE_CAN_MESSAGE(0x721)[DATA_INDEX])
+}
+----
+
+== Charge Test
+----
+Reading (Gpio Pin PC6)
+Within 100ms{
+    Expect (CHARGE_CAN_MESSAGE(0x721)[DATA_INDEX])
+    LED Common Contactor ON
+    LED Safety Contactor ON
+    LED Charge Contactor ON
+}
+----
+
+== Discharge Test
+----
+Reading (Gpio Pin PC7)
```

Comment:
There is no action In these tests to initiate a change. Is this supposed to be `Turn GPIO Pin PC7 On`?

Reply:
It's supposed to read the CHARGE, DISCHARGE, and SAFETY inputs from the Orion BMS, what's the best way to represent that?

Reply:
How does it read them? Generally in tests, most of the code is initiators and you only have one or a few lines that are checking something (the expects). Reading isn't an initiator, so I'd expect there to be a mock "write" here or something like that.